src/modules/word/index.ts

```diff
@@ -596,6 +596,7 @@ export class WordModule {
    *
    * @param options The optional options object or the number of words to return.
    * @param options.count The number of words to return. Defaults to a random value between `1` and `3`.
+   * @param options.capitalize Capitalize the first letter of each word. Defaults to `none`
```

Comment:
```suggestion
   * @param options.capitalize Capitalize the first letter of each word. Defaults to `'none'`.
```

We should explain which options exists (for the non TS users) and what each option does.

Reply:
sure, I will add a brief description in the JSDoc and change the code bits according to your suggested changes